tests/rkt_tests.go

```diff
@@ -124,11 +124,19 @@ func spawnOrFail(t *testing.T, cmd string) *gexpect.ExpectSubprocess {
 	return child
 }
 
+// waitOrFail waits for the child to exit, draining all its output.
+// If a non-negative return value is provided, child exit status must match.
 func waitOrFail(t *testing.T, child *gexpect.ExpectSubprocess, expectedStatus int) {
+	bufOut := []string{}
+	ttyIn, ttyOut := child.AsyncInteractChannels()
+	close(ttyIn)
```

Comment:
:+1: Can we leave a TODO to change `gexpect.AsyncInteractChannels` to accept channels rather than returning them?